tests/test_helpers/helpers.ts

```diff
@@ -324,8 +324,12 @@ export function target(str: string): Zone[] {
   return str.split(",").map(zone);
 }
 
+export function toSingleRangeData(sheetId: UID, xc: string): RangeData {
```

Comment:
```suggestion
export function toRangeData(sheetId: UID, xc: string): RangeData {
```
I know the name would be very similar to `toRangesData` but "single" feels meh